### ISSUE TITLE
TFMA fails when trying to use a custom metric from a module that ends with `keras.metrics`

### ISSUE THREAD

**Issue by rafaelbarreto87** [tensorflow/model-analysis#76]
### System information

-   **Have I written custom code (as opposed to using a stock example script
    provided in TensorFlow Model Analysis)**: Yes
-   **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: `tensorflow/tensorflow:2.2.0-gpu-jupyter` docker image
-   **TensorFlow Model Analysis installed from (source or binary)**: binary
-   **TensorFlow Model Analysis version (use command below)**: 0.22.2
-   **Python version**: 3.6.9
-   **Jupyter Notebook version**: 6.0.2
-   **Exact command to reproduce**: NA

### Describe the problem

TFMA fails when trying to use a custom metric coming from a module that ends with `keras.metrics`. I have a contrib module whose name is `{PROJECT_NAME}.ml_utils.tf.keras.metrics` in which I have lots of custom metrics. When I try to use those in TFMA, it fails saying that the metrics can't be found. I _don't_ have this issue if I rename my module to something like `{PROJECT_NAME}.ml_utils.tf.keras.custom_metrics`. After tracking the issue down, I noticed [this snippet](https://github.com/tensorflow/model-analysis/blob/0b10d4175235cf68d5364282a862ed02151c3b1e/tensorflow_model_analysis/metrics/tf_metric_wrapper.py#L266) to be the issue:

```
def _custom_objects(
    metrics: Dict[Text, List[tf.keras.metrics.Metric]]) -> Dict[Text, Any]:
  custom_objects = {}
  for metric_list in metrics.values():
    for metric in metric_list:
      if (not metric.__class__.__module__.endswith('keras.metrics') and
          not metric.__class__.__module__.endswith('keras.losses')):
        custom_objects[metric.__class__.__module__] = metric.__class__.__name__
  return custom_objects
```

The if statement is problematic because my module indeed ends with `keras.metrics` and, yet, is a custom module.  It seems the more correct and robust approach here would be to replace it with:

```
def _custom_objects(
    metrics: Dict[Text, List[tf.keras.metrics.Metric]]) -> Dict[Text, Any]:
  custom_objects = {}
  for metric_list in metrics.values():
    for metric in metric_list:
      if (not metric.__class__.__module__ == tf.keras.metrics.__name__ and
          not metric.__class__.__module__ == tf.keras.losses.__name__):
        custom_objects[metric.__class__.__module__] = metric.__class__.__name__
  return custom_objects
```

**Comment by mdreves** [comment]
Thanks for raising the issue. Should be fixed at head now.

**Comment by rafaelbarreto87** [comment]
That was quick! Thanks :D.
